src/main/telemetry/telemetry.h

```diff
@@ -54,7 +54,7 @@ typedef struct telemetryConfig_s {
 
 PG_DECLARE(telemetryConfig_t, telemetryConfig);
 
-#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_MAVLINK)
+#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_MAVLINK | FUNCTION_TELEMETRY_IBUS)
```

Comment:
At the moment it is: https://github.com/betaflight/betaflight/blob/master/src/main/fc/fc_init.c#L448-L480

I think there are other protocols like CRSF and JetiExBus that have the same requirements, so this is unlikely to change.